### ISSUE TITLE
TAP driver should print logs for failed tests

### ISSUE THREAD

**Issue by LukeShu** [datawire/build-aux#5]
This would require teaching the TAP-emitters to include the logs as proper YAML, instead of just comments? And then teaching `tap-driver summarize` to print that.

**Comment by rhs** [comment]
Debugging CI test failures is quite difficult because the test suite is growing, and the go test runner runs stuff in parallel by default. This results in lots of interleaved output, and the logs can be super hard to follow. We are at the point where we really need to have a single linear log per sequential context.
